wiki/Tournaments/GTS/AGTS_2019/en.md

```diff
@@ -0,0 +1,373 @@
+| ![][flag_ES] | **Spain** | **[naliem](https://osu.ppy.sh/users/3252709)**, [RuinedChicken](https://osu.ppy.sh/users/7983713), [Machango](https://osu.ppy.sh/users/7180552), [Nara\_NB](https://osu.ppy.sh/users/3593992) |
+| ![][flag_TW] | **Taiwan** | **[- DeveRUX -](https://osu.ppy.sh/users/2900875)**, [Hatsuse](https://osu.ppy.sh/users/697007), [nw9v4g46](https://osu.ppy.sh/users/2169329), [david649713](https://osu.ppy.sh/users/1390823) |
+| ![][flag_GB] ![][flag_PT] | **United Kingdom/Portugal** | **[Metzo](https://osu.ppy.sh/users/10633982)**, [w1ll](https://osu.ppy.sh/users/11406987), [Ali161102](https://osu.ppy.sh/users/7582420), [Yumi\_Hideaki](https://osu.ppy.sh/users/10081162) |
+| ![][flag_US] | **United States A** | **[Backfire](https://osu.ppy.sh/users/263110)**, [ChurroChef](https://osu.ppy.sh/users/9258564), [CaptainEChan](https://osu.ppy.sh/users/9536977), [dragonworm](https://osu.ppy.sh/users/11464752) |
+| ![][flag_US] | **United States B** | **[driodx](https://osu.ppy.sh/users/9709548)**, [Shindamama](https://osu.ppy.sh/users/12780046), [R J](https://osu.ppy.sh/users/6490509), [DiggyDogg](https://osu.ppy.sh/users/5014664) |
+| ![][flag_VE] | **Venezuela** | **[Nutellitah](https://osu.ppy.sh/users/10613075)**, [JessiChan](https://osu.ppy.sh/users/1368025) |
+
+## Groups
+
+| Group | Top Seed | High Seed | Low Seed | Unseeded |
+| :-: | :-- | :-- | :-- | :-- |
+| **A** | ![][flag_PL] Poland | ![][flag_RU] Russian Federation B | ![][flag_PH] Philippines | ![][flag_MY] Malaysia |
+| **B** | ![][flag_JP] Japan | ![][flag_AR] Argentina | ![][flag_HK] Hong Kong | ![][flag_CL] Chile |
+| **C** | ![][flag_NL] Netherlands | ![][flag_FR] France | ![][flag_MX] Mexico | ![][flag_TW] Taiwan |
+| **D** | ![][flag_US] United States A | ![][flag_RU] Russian Federation A | ![][flag_US] United States B | ![][flag_BR] Brazil |
+| **E** | ![][flag_KR] South Korea | ![][flag_DE] Germany A | ![][flag_DK] ![][flag_NO] ![][flag_SE] Scandinavia | ![][flag_VE] Venezuela
+| **F** | ![][flag_CA] Canada | ![][flag_GB] ![][flag_PT] United Kingdom/Portugal | ![][flag_ID] Indonesia A | ![][flag_ID] Indonesia B |
+
+---
+
+## Mappools
+
+### Quarterfinals
+
+**[Download the map pack here! (83 MB)](https://mega.nz/#!spkR0agQ!MokudfOpqItg9GU4KnYaD-jRjMLVYctMnBeRmWDtGDA)**
+
+- NoMod
+  - [Function Phantom - Euclid (salchow) \[Green's Fifth Axiom\]](https://osu.ppy.sh/beatmapsets/941145#taiko/2002626)
+  - [S.S.H. - BassTest (Ak1o) \[TEST\]](https://osu.ppy.sh/beatmapsets/666764#taiko/1411059)
+  - [Chroma - I (rubies87) \[Oni\]](https://osu.ppy.sh/beatmapsets/883570#taiko/1933318)
+  - [Camellia - \*Feels Seasickness...\* (Genjuro) \[Inner Oni\]](https://osu.ppy.sh/beatmapsets/1014457#taiko/2126517)
+  - [Sadkey - Further (Airuei Redo) (Nifty) \[Discovery\]](https://osu.ppy.sh/beatmapsets/741764#taiko/1564820)
+- EX
```

Comment:
explain in parenthesis what this means, the only mention of "EX" further down (in an unrelated section of the rules) says that EX is for harder maps but I still don't know what mods are allowed